pin/models.py

```diff
@@ -49,6 +49,7 @@ class Pin(models.Model):
     status=models.CharField(max_length=20, choices=STATUS, default="valid")
     pin = models.IntegerField( blank=True, null=True, editable=False)
     caller = models.ForeignKey(Caller, on_delete=models.CASCADE, null=True, blank=True)
+    location = models.CharField(max_length=200, null=True, blank=True)
```

Comment:
Please change the max_length to 256
Memory allocated to an unknown resources should be in mutiples of 8

Reply:
Alright sir